homeassistant/components/updater/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 CONFIG_SCHEMA = vol.Schema(
     {
-        DOMAIN: {
+        vol.Optional(DOMAIN, default={}): {
```

Comment:
It's still an empty dict with no default values here. The validator won't run on the default value on the keys.

You can separate the value schema out and set the default like: default=VALUE_SCHEMA({})

Reply:
The validator *will* run on default values.
```
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import voluptuous as vol

In [2]: import homeassistant.helpers.config_validation as cv

In [3]: CONF_REPORTING = "reporting"
   ...: CONF_COMPONENT_REPORTING = "include_used_components"
   ...: 
   ...: DOMAIN = "updater"
   ...: 
   ...: CONFIG_SCHEMA = vol.Schema(
   ...:     {
   ...:         vol.Optional(DOMAIN, default={}): {
   ...:             vol.Optional(CONF_REPORTING, default=True): cv.boolean,
   ...:             vol.Optional(CONF_COMPONENT_REPORTING, default=False): cv.boolean,
   ...:         }
   ...:     },
   ...:     extra=vol.ALLOW_EXTRA,
   ...: )

In [4]: CONFIG_SCHEMA({})
Out[4]: {'updater': {'include_used_components': False, 'reporting': True}}
```

Reply:
Okey. That was news to me. Either i have missed that or it was changed. Good to know.

Reply:
Ok. Was added in 0.11.1. I knew i wasn't just crazy :)

Reply:
Yep, three years old news. :smile: